### ISSUE TITLE
setup.py uncomplet

### ISSUE THREAD

**Issue by foxmask** [halcy/Mastodon.py#73]
when running test, there are missing lib ; so seetup.py could be like this 

```python
from setuptools import setup

install_requires = [
    'pytest==3.2.1',
    'python-dateutil==2.6.1',
    'pytz==2017.2',
    'requests==2.18.4',
    'six==1.10.0',
]

setup(name='Mastodon.py',
      version='1.0.8',
      description='Python wrapper for the Mastodon API',
      packages=['mastodon'],
      setup_requires=['pytest-runner'],
      tests_require=['pytest'],
      install_requires=install_requires,
      url='https://github.com/halcy/Mastodon.py',
      author='Lorenz Diener',
      author_email='lorenzd+mastodonpypypi@gmail.com',
      license='MIT',
      keywords='mastodon api microblogging',
      classifiers=[
          'Development Status :: 5 - Production/Stable',
          'Intended Audience :: Developers',
          'Topic :: Communications',
          'License :: OSI Approved :: MIT License',
          'Programming Language :: Python :: 2',
          'Programming Language :: Python :: 3',
      ]
)
```

You could also provide a requirements.txt too that will contain all that lib

**Comment by Elizafox** [comment]
This isn't the right fix.

You want python-dateutil.

**Comment by halcy** [comment]
Yep, that was a heckup on my part. It was originally (wrongly) set to "dateutils", I changed it to "dateutil". Will fix.

**Comment by halcy** [comment]
Fixed by #77
